packages/orchestrator/internal/sandbox/uffd/userfaultfd/cross_process_helpers_test.go

```diff
@@ -332,6 +332,8 @@ func crossProcessServe() error {
 	signal.Notify(exitSignal, syscall.SIGUSR1)
 	defer signal.Stop(exitSignal)
 
+	readyFile := os.NewFile(uintptr(6), "ready")
```

Comment:
The FD offsets are ugly, but I'm not sure there's a simple way to document "this is the input index + 3" =/ if you can think of one, it'd be really good to have here!